ChatQnA/benchmark_results.md

```diff
@@ -0,0 +1,161 @@
+# ChatQnA Benchmark Results
+
+## Overview
+
+ChatQnA deployed on a single node with ICX cores as the head node and supporting 8x Gaudi2 cards.
+This is based on OPEA v1.3 release helm charts and images using vLLM inferencing platform.
+
+## Methodology
+
+Tests scale concurrent users from 1 to 256, and each user send 4 queries. Measuring end to end (E2E) latency average for each query, time to first token (TTFT) average and time per output token (TPOT) average.
```

Comment:
Would be good to specify whether TPOT includes first token or not.

(Benchmark sends only 1x4 - 256x4 queries?   I.e. more of a "cold" KV cache case, than "hot" one?)